src/HelloIntelliJ.java

```diff
@@ -1,7 +1,12 @@
+import java.util.Scanner;
 public class HelloIntelliJ {
     public static void main(String[] args) {
-        System.out.println("Hello IntelliJ! This is Moin");
-    }
-}
-
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter Your Name:");
+        String message = scanner.nextLine();
+        // add line to display message
+// add line to close scanner }}
+        scanner.close();
+        System.out.println(message);
```

Comment:
please have proper order of commands, 
close should be at the end

Reply:
Done. Thank You.

Reply:
its ok, thanks